.github/workflows/cd.yml

```diff
@@ -61,61 +63,41 @@ jobs:
         if not is_release and upload_to == "app-store":
           upload_to_apple = False
 
-        extra_xcode = os.getenv("EXTRA_XCODEBUILD", "")
-        if os.getenv("PLATFORM") == "iOS":
-          extra_xcode += f" -authenticationKeyPath {os.getenv('APPLE_STORE_AUTH_KEY_PATH')}"
-          extra_xcode += f" -authenticationKeyID {os.getenv('APPLE_STORE_AUTH_KEY_ID')}"
-          extra_xcode += f" -authenticationKeyIssuerID {os.getenv('APPLE_STORE_AUTH_KEY_ISSUER_ID')}"
-
         with open(os.getenv("GITHUB_ENV"), "a") as fh:
           fh.write(f"VERSION={version}\n")
           fh.write(f"ISRELEASE={'yes' if is_release else ''}\n")
           fh.write(f"EXPORT_METHOD={export_method}\n")
           fh.write(f"UPLOAD_FOLDER={upload_folder}\n")
-          fh.write(f"EXTRA_XCODEBUILD={extra_xcode}\n")
           fh.write(f"UPLOAD_TO_APPLE={'yes' if upload_to_apple else ''}\n")
 
     - name: Prepare use of Developper ID Certificate
-      if: ${{ matrix.destination.uploadto == 'dmg' }}
-      shell: bash
+      if: matrix.destination.uploadto == 'dmg'
```

Comment:
Please harmonize with CI. You changed `dmg` to `ftp` there for some reason so use same naming in both workflows

Reply:
Please see it updated.